framework/src/modules/chain/submodules/blocks.js

```diff
@@ -20,7 +20,7 @@ const { BLOCK_RECEIPT_TIMEOUT, EPOCH_TIME } = global.constants;
 const {
 	CACHE_KEYS_BLOCKS,
 	CACHE_KEYS_TRANSACTIONS,
-} = require('../../../../../framework/src/components/cache');
+} = require('../../../components/cache');
```

Comment:
I am not sure if we keep these or not, as its easily accessible by `scope.components.cache.CACHE_KEYS_BLOCKS`. But probably refactor this separately.